client/src/components/J40Map.tsx

```diff
@@ -1,29 +1,28 @@
 /* eslint-disable no-unused-vars */
 import React, {useRef, useEffect, useState} from 'react';
-import {LngLatBoundsLike,
+import maplibregl, {LngLatBoundsLike,
   Map,
   NavigationControl,
   PopupOptions,
   Popup,
-  LngLatLike} from 'mapbox-gl';
+  LngLatLike} from 'maplibre-gl';
+import 'maplibre-gl/dist/maplibre-gl.css';
 import mapStyle from '../data/mapStyle';
 import ZoomWarning from './zoomWarning';
 import PopupContent from './popupContent';
 import * as constants from '../data/constants';
 import ReactDOM from 'react-dom';
-import 'mapbox-gl/dist/mapbox-gl.css';
-import * as styles from './mapboxMap.module.scss';
+import * as styles from './J40Map.module.scss';
 
 declare global {
   interface Window {
     Cypress?: object;
-    mapboxGlMap: Map;
+    underlyingMap: Map;
   }
 }
+type ClickEvent = maplibregl.MapMouseEvent & maplibregl.EventData;
 
-type ClickEvent = mapboxgl.MapMouseEvent & mapboxgl.EventData;
-
-const MapboxMap = () => {
+const J40Map = () => {
```

Comment:
I like making the map variables more generic like this (e.g. not associated with a particular map impl).